Unity-Technologies/Graphics
# [XPipeline]Move Decal Gizmo color Register on Preferences/Colors

### Purpose of this PR
Use the shared method to register Preferences Colors.

---
### Testing status

- Check that the Decal Gizmo color is correctly registered also if the Preferences/Colors even if the panel is opened at StartUp.
- Check that the color of the gizmo is updated properly and reset to the default value

---
### Comments to reviewers
Simply move @RSlysz  code to the shared `RegisterPreferenceColor` from the `CoreRenderPipelinePreferences`
